docs/modules/operation/pages/events/sources/snmp-traps.adoc

```diff
@@ -16,26 +16,27 @@ By default, {page-component-title} listens on a non-standard port for trap messa
 . Ensure that SNMP-capable devices on your network are configured to send traps to {page-component-title}.
 
 [[trap-config]]
-== SNMP trap configuration: how to
+== SNMP trap definition
 
-Configure SNMP traps to customize the types of information you receive and the conditions under which {page-component-title} converts the trap into an event.
+You can configure SNMP traps to customize the types of information you receive and the conditions under which {page-component-title} converts traps into an event.
 For example, you may want to see an event only when a device is down, or when a combination of criteria are met (for example, OID X with "major" severity level and status "indeterminate").
 These traps are transformed into events only when the conditions you define are met.
 
 There are several ways to configure {page-component-title} to customize the events you receive and see:
 
 * Determine what problem you are trying to solve (for example, "I want to know X about my Cisco router").
-* Manually send the corresponding event to the {page-component-title} server and see how the event is handled (see xref:operation:events/sources/send-event.adoc[Manually send events]).
-* Make changes to the xref:operation:admin/webui/mib.adoc[SNMP MIB compiler]).
+* Manually send the corresponding event to the {page-component-title} server and see how the event is handled (see xref:operation:events/event-debugging.adoc#send-event[Manually send events]).
+* Import event definitions with the xref:operation:admin/webui/mib.adoc[SNMP MIB compiler].
 * Configure the event to meet your requirements, if required.
 ** <<trap-def-create, Create trap definitions>> to specify criteria for when {page-component-title} creates an event.
-** <<varbind-filter,Filter on variable bindings>> for further granularity on the conditions for creating an event.
+** <<varbind-filter, Filter on variable bindings>> for further granularity on the conditions for creating an event.
 
 [[trap-def-create]]
 === Create event definition for an SNMP trap
+
 {page-component-title} includes event definitions for traps from many vendors' equipment.
 Use the `mask` tag to match SNMP traps in `eventconf.xml`.
-See xref:operation:events/anatomy-events.adoc#ga-events-anatomy-of-an-event[Anatomy of an event] for details on event structure.
+See xref:operation:events/event-definition.adoc#ga-events-anatomy-of-an-event[Anatomy of an event] for details on event structure.
 
 The following example shows a Cisco Systems event for their C3800 device.
 Parts of it look similar to internally generated events, with the main difference being the `mask` block.
```

Comment:
Could you delete lines 88-91 below? It was supposed to be commented out, but appears in the built version of the docs. I never could find that information from Cisco.